### DISCUSSION TITLE
2fa codes lost "Can't access to the account"

### DISCUSSION THREAD

**Discussion by bakheet22** [community/community#23094] [New to GitHub]
<p>Hello there</p>
<p>I need a help in this issue.</p>
<p>I just lost my phone and it had all the login data and codes, now I am trying to login to github with out any success.</p>
<p>I tried all the steps to prove it’s mine till I reached this screen</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/f/2f57301d557bfca4ca64d10ee6e08a2287627329.png" data-download-href="/uploads/short-url/6KNeUcz0lQJ84wpO5Bf2fAHukMV.png?dl=1" title="Capture" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/89732649/181074820-aa8ba680-d2f0-4505-b999-87713f8aa16a.png" alt="Capture" data-base62-sha1="6KNeUcz0lQJ84wpO5Bf2fAHukMV" width="468" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/f/2f57301d557bfca4ca64d10ee6e08a2287627329_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture</span><span class="informations">539×575 19.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>it says “Cannot verify this device”</p>
<p>and I don’t know what is personal access token.</p>
<p>I have no data saved in third party or something.</p>
<p>what should I do?</p>

**Comment by mpboom** [comment, marked as answer]
<p>Hi and welcome <a class="mention" href="/bakheet22">@bakheet22</a> . On the forums we cannot discuss account-related matters. If you’ve lost access to your GitHub account, please reach out to GitHub Support:<br>
<a href="https://git.io/c" class="onebox" target="_blank" rel="noopener nofollow ugc">https://git.io/c</a></p>

**Comment by adithyare** [comment]
<p>I have the similar problem. OP did support restore your account access?</p>
